.github/problem-matcher.json

```diff
@@ -0,0 +1,18 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "maven",
```

Comment:
owner seems to be reverted back?

Reply:
Huh, it must have got lost in one of the rebase I did. Thanks for staying vigilant, fixed!